src/states/Preroll.js

```diff
@@ -227,16 +227,21 @@ class Preroll extends AdState {
   }
 
   resumeAfterNoPreroll(player) {
+
     // Resume to content and unblock play as there is no preroll ad
     this.contentResuming = true;
     player.ads._shouldBlockPlay = false;
+    this.cleanupPartial(player);
```

Comment:
This seems to address the player being placed in a bad state (e.g. loading spinner) in some situations.